gdraw/gresource.c

```diff
@@ -168,12 +168,7 @@ return;
     else
 return;
 
-    free(GResourceProgramDir);
-    GResourceProgramDir = _GFile_find_program_dir(prog);
-    if ( GResourceProgramDir==NULL ) {
-	GFileGetAbsoluteName(".",filename,sizeof(filename));
-	GResourceProgramDir = copy(filename);
-    }
+    FindProgDir(prog);
```

Comment:
Above in this file is the line:
```
char *GResourceProgramName, *GResourceFullProgram, *GResourceProgramDir;
```
Then in fsys.c is the line:
```
char *GResourceProgramDir = NULL;
```
Which is "new" but only because it was moved from a different place in the file and the old 0 replaced with NULL.

So unless I'm missing something it appears that FontForge has two non-static `GResourceProgramDir` globals.

That already puts me a bit out of my depth as to how things work, but this change makes it even more confusing, as it appears that the manipulation of this file's `GResourceProgramDir` is being replaced with the manipulation of the other file's `GResourceProgramDir`, which is then somehow being used constructively below.

Or maybe there is only one because the two declarations get implicitly merged? 

Anyway, even if all this does work wouldn't it be preferable to replace the instances of `GResourceProgramDir` in this file with calls to `getGResourceProgramDir()`?

Reply:
Good spot on the duplication, I missed that.

I've opted to just remove all usages of it.